### DISCUSSION TITLE
Diskussion zu den Ergebnissen

### DISCUSSION THREAD

**Discussion by samueltrippler** [malangmann/GitHub-Aufgabe_Gruppe1#7] [General]
Hey, Leute!

Lasst uns hier die Auswertung der Daten diskutieren. Mir ist aufgefallen, dass es bei Interesse an Mathematik / Programmieren keine Werte geringer als 3 gibt. 

In dem Plot, den Marius hochgeladen hat, sieht man, dass 100% der Statistiker und 100% der Mathematiker Mathe-LK hatten, während es bei den Data-Science-Leuten und den Informatikern nur ca. 50% sind.

Man sieht auch, dass fünf von sechs Mathematikern Interesse an Mathematik mit 7 bewertet haben. Statistiker und Informatiker haben im Vergleich mit den Data-Science-Leuten ein höheres Interesse an Mathematik.
Beim Interesse an Programmieren haben die Date-Science-Studenten und die Informatiker die höchsten Werte. Die Statistiker scheinen das geringste Interesse zu haben.

**Comment by tmmmarius** [comment]
In der Funktion von 3b), kann man sehen das der relative Anteil an Statistikern und Informatikern fast gleich ist, wobei die Wahrscheinlichkeit für Informatik- und Mathematikstudenten etwas geringer ist. Es wäre interessant zu wissen wie groß der Unterschied der Wahrscheinlichkeiten für Statistiker und Informatikern ist.

**Reply by malangmann** [reply]
Die Studienfächer wurden wie auf dem Aufgabenblatt beschrieben mit gleichen W'keiten für Statistik und Data Science, Informatik mit einer etwas geringerer Wahrscheinlichkeit und Mathe mit der geringsten Wahrscheinlichkeit, gesampelt. Das sich das in den simulierten Daten etwas anders darstellt, hat mich eben auch etwas gewundert als ich mir Eure Grafik hier im Repository angeschaut habe. 35% der Studenten studieren jeweils Data Science und Statistik, 20% Informatik und 10% Mathe.

**Comment by malangmann** [comment]
moin, 

die Beobachtung dass bei Interesse Mathematik /Programmieren keine Werte geringer als 3 sind stimmt. Die Aufgabe wurde hier so interpretiert, dass die Werte Elemente der Menge 1-7 sein sollen. Jedoch wurde angenommen, das niemand mit einem Interesse von 1 oder 2 eines der Fächer studiert. Außer vllt. für das Semesterticket, aber die seien hier mal außen vor.

Mathe-LK wurde in Abhängigkeit von einer der numerischen Variablen Interesse Mathe oder Interesse Informatik vergeben. Das als Hinweis, falls ihr da nochmal schauen wollt. 

Statistiker und Informatiker haben ein etwas größeres Interesse an Mathe als Data-Scientisten. Sogar das Gleiche.  stimmt also. sehr gut.

Angenommen wurde, dass Statistiker und Mathematiker gleich wenig Interesse an Programmieren haben. Data Scientisten habe schon deutlich mehr Interesse an Programmieren. Nicht so viel wie Informatiker, denn die lieben programmieren. 

Beim Interesse an Mathematik von Mathematikern, wurden nur die Interessensstufen 5, 6 und 7 vergeben mit den W'keiten 20%, 40%, 40%. Dadurch das er nur 6 Mathematiker gibt, scheint sich zufällig diese Verteilung ergeben zu haben. Eure Interpretation der Daten ist richtig.

**Reply by tmmmarius** [reply]
Ist ja interessant, da man bei der Interesse für Programmieren bei den Data Scientisten keine klare Tendenz erkennen kann, ich dachte erstmal deren Interesse für Programmieren wäre eher mittelmäßig. War das der Zufall oder ist der von euch gewählte Zusammenhang dem ähnlich?

**Reply by samueltrippler** [reply]
Wenn man für die Data-Scientist-Studenten das arithmetische Mittel bezüglich des Interesses an Programmieren berechnet, dann kommt man auf: (36 + 50 + 66 + 63)/39 = 5.51
Für die Statistiker: 112/28 = 4
Für Mathematiker: 27/6 = 4.5
Für Informatiker: 6.48

Das kommt ja mit dem, was Markus gesagt hat, ungefähr hin.

**Reply by samueltrippler** [reply]
OK, anscheinend hatten alle Mathe-LK, die bei Interesse an Mathe mehr als 3 angegeben haben.
Das haben wir nicht in den Funktionen, aber habe mir einfach mal
dat$InteresseMathe[which(dat$MatheLK == 0)]
dat$InteresseMathe[which(dat$MatheLK == 1)]
angeschaut und festgestellt, dass es beim ersten nur Dreien gibt und beim zweiten keine Dreien.

**Reply by tmmmarius** [reply]
Ja aber wenn du einen Data Scientist aus unserem Datensatz auswählst, ist die Chance beinahe 50/50, dass dieser ein großes Interesse an Programmieren hat (ich sage jetzt einfach mal ein großes Interesse ist gleich 6 oder 7) oder ein "mittelgroßes" Interesse (= 4 oder 5). Ich hatte den Gedanken, da ich dachte, dass man eher welche findet, die ein großes Interesse für Programmieren haben.

**Reply by samueltrippler** [reply]
Ja, für die Data-Scientists haben sie alles von 4 bis 7 zugelassen, deshalb gibt es keine klare Tendenz.

**Reply by malangmann** [reply]
Genau richtig. Alle deren Interesse an Mathe größer 3 ist für die wurde angenommen, dass sie auch Mathe-LK hatten.
